mm2src/mm2_main/src/lp_ordermatch/best_orders.rs

```diff
@@ -286,13 +291,14 @@ pub async fn best_orders_rpc(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>,
 pub enum BestOrdersRpcError {
     CoinIsWalletOnly(String),
     P2PError(String),
+    CtxError(String),
 }
 
 impl HttpStatusCode for BestOrdersRpcError {
     fn status_code(&self) -> StatusCode {
         match self {
             BestOrdersRpcError::CoinIsWalletOnly(_) => StatusCode::BAD_REQUEST,
-            BestOrdersRpcError::P2PError(_) => StatusCode::INTERNAL_SERVER_ERROR,
+            _ => StatusCode::INTERNAL_SERVER_ERROR,
```

Comment:
It's better to explicitly mention all errors here so that adding more error variant to `BestOrdersRpcError` will cause a compile error that has to be handled correctly here.
```suggestion
            BestOrdersRpcError::P2PError(_) | BestOrdersRpcError::CtxError(_) => StatusCode::INTERNAL_SERVER_ERROR,
```

Reply:
[done](https://github.com/KomodoPlatform/atomicDEX-API/pull/1849/commits/4f1f5c27df552063ca9e0df8181da192c9e7266f)